tests/x11/libcamgm.pm

```diff
@@ -19,7 +19,7 @@ my $password = $testapi::password;
 my $email    = "you\@example.com";
 
 sub restart_yast2_camgm {
-    assert_and_click("xterm");
+    assert_and_click("xterm-selected");
     type_string("yast2 ca_mgm\n");
```

Comment:
Consider launch_yast2_module_x11: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/y2x11test.pm#L16 in future to run yast modules